src/container/RelayRenderer.js

```diff
@@ -123,10 +124,17 @@ class RelayRenderer extends React.Component {
     super(props, context);
     this.mounted = true;
     this.state = this._runQueries(this.props);
+
+    this.props.relayContext.injectBatchingStrategy(
+      ReactDOM.unstable_batchedUpdates
+    );
```

Comment:
But the later approach have a problem that only the default instance will have the batching strategy injected. Injecting `ReactDOM.unstable_batchedUpdates` to every new `RelayContext` by default is also bad, because in couples Relay to ReactDOM too much.

I think, that could be solved by subclassing `RelayContext` with `ReactDOMRelayContext`, so only the later will inject `ReactDOM.unstable_batchedUpdates`. And make `ReactDOMRelayContext` the default for the `context` property of `RelayRenderer`:
```javascript
RelayRenderer.defaultProps = {
  context: ReactDOMRelayContext.getDefaultInstance(),
}
```
And make `RelayStore` an alias of `ReactDOMRelayContext.getDefaultInstance()` for backward compatibility.

cc @josephsavona